src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public abstract class Figure implements Drawable, GetArea {
+    private Color color;
```

Comment:
Add a constructor with a parameter to be able to set this variable in the children's constructors.